pom.xml

```diff
@@ -15,15 +15,6 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <slf4j.version>1.7.26</slf4j.version>
-        <log4j.version>1.2.17</log4j.version>
-        <commons-compress.version>1.18</commons-compress.version>
-
-        <!-- for the correct jackson and guava versions see https://github.com/dropwizard/dropwizard/blob/release/1.3.x/dropwizard-bom/pom.xml -->
```

Comment:
Is there a reason you added this link to the bom file, but did not include the bom file here? It provides the correct version for jackson, guava and many more (maybe too many and it would be more readable to specify these versions explicitly in our dependency management section instead?)